.github/workflows/draft-release.yml

```
@@ -0,0 +1,35 @@
+name: draft-release
+
+on:
+  pull_request: ~
+  push:
+    tags: ['v*']
+
+jobs:
+  draft-github-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v3
+        with:
+          version: ${{ vars.PNPM_VERSION }}
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ vars.DEFAULT_NODE_VERSION }}
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'pnpm'
+
+      - name: install and build
+        run: pnpm install --frozen-lockfile
+      # TODO: Reuse step for GA release
+      # - uses: sap/cloud-sdk-js/.github/actions/get-changelog@main
+      #   name: Get Changelog
+      #   id: get-changelog
+      - uses: ncipollo/release-action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          draft: true
+          tag: test
```

[req] needs to be removed before merge!!!

```suggestion
          tag: test
```